frontend/lib/pyodide/runtime.ts

```
@@ -0,0 +1,249 @@
+    for _fig_num in plt.get_fignums():
+        _fig = plt.figure(_fig_num)
+        _buf = io.BytesIO()
+        _fig.savefig(_buf, format='png', bbox_inches='tight', dpi=96)
+        _buf.seek(0)
+        _plots.append('data:image/png;base64,' + base64.b64encode(_buf.read()).decode('utf-8'))
+        plt.close(_fig)
+
+[_stdout_capture.getvalue(), _stderr_capture.getvalue(), _plots]
+`;
+
+    let result: ExecutionResult = {
```

ESLint

[error] 'result' is never reassigned. Use 'const' instead.

(prefer-const)